src/main/java/com/idrsolutions/microservice/utils/ZipHelper.java

```diff
@@ -82,11 +95,10 @@ private static void addFileToZip(final String path, final String srcFile, final
     }
 
     /**
-     * Zips the given folder (including all sub files and writes to
-     * ZipOutputStream.
+     * Zips the given folder (including all sub files and writes to ZipOutputStream.
      *
-     * @param path The path to the folder to recursively zip
-     * @param srcFolder the name of the folder to zip
+     * @param path the current location within the zip file
```

Comment:
@leonatherton The code changes look good. One minor point if the Javadoc is being updated, should the path param value give the same warning as the addFileToZip Javadoc? (given that path is passed into that method unchanged).

```suggestion
     * @param path the current location within the zip file (must include a trailing slash unless the value is empty)
```